src/app.js

```diff
@@ -396,11 +398,10 @@ app.route("/dist/*")
  * instead of served straight from disk.
  */
 if (app.locals.gatsbyDevUrl) {
-  // eslint-disable-next-line global-require, import/no-extraneous-dependencies
-  const {createProxyMiddleware} = require("http-proxy-middleware");
+  const {createProxyMiddleware} = await import("http-proxy-middleware");
```

Comment:
After some ESLint upgrading/tinkering, there's still [this error](https://github.com/nextstrain/nextstrain.org/runs/8239083567?check_suite_focus=true#step:7:11):

```
  401:34  error  Parsing error: Can not use keyword 'await' outside an async function (401:34)
```

Supposedly, top-level await should work with ESLint 8...

- https://github.com/eslint/eslint/issues/14632#issuecomment-908795546
- https://github.com/eslint/eslint/discussions/13934
- https://eslint.org/docs/latest/user-guide/configuring/language-options

Will investigate more another day.

Reply:
After playing in a [test repo](https://github.com/victorlin/nodejs-test-apps), I deduced that the Babel parser used in this project also has an affect on supporting top-level await. Notably, it can work with earlier versions of ESLint as long as the Babel parser is at least v7.15.0 based on [the changelog entry](https://github.com/babel/babel/blob/3057a3b2f7782993a293aeb0ef629b153ac4422f/CHANGELOG.md#v7150-2021-08-04).

The reason this error persisted is because 4fd47d425a77b27ddf45a82047152baf87b58c27 upgraded the Babel parser but did not update Babel itself. After doing that in 177b0aefa2c327b0c28f677776ed3b787cb99421, ESLint passes but there are now build issues with Auspice's dependency pinning not supporting the latest version of Babel. Need to dig deeper to fix that, but at least this is making sense now.

Reply:
Moving the tangent issue to https://github.com/nextstrain/nextstrain.org/pull/583#discussion_r966353409